doc/cephfs/fs-volumes.rst

```diff
@@ -99,32 +99,11 @@ expected to be disabled on the volume.
 
 Fetch the information of a CephFS volume using::
 
-    $ ceph fs volume info vol_name
-    {
-        "mon_addrs": [
-            "192.168.1.7:40977"
-        ],
-        "pending_subvolume_deletions": 0,
-        "pools": {
-            "data": [
-                {
-                    "avail": 106288709632,
-                    "name": "cephfs.vol_name.data",
-                    "used": 4096
-                }
-            ],
-            "metadata": [
-                {
-                    "avail": 106288709632,
-                    "name": "cephfs.vol_name.meta",
-                    "used": 155648
-                }
-            ]
-        },
-        "used_size": 0
-    }
+    $ ceph fs volume info vol_name [--human_readable]
 
-The output format is json and contains fields as follows.
+'--human_readable' flag allows the command to show the used/avail pool sizes in KB/MB/GB.
```

Comment:
The \`\`--human_readable\`\` flag shows used and available pool capacities in KB/MB/GB.

Reply:
Done.